tests/main/snapd-snap/task.yaml

```diff
@@ -144,6 +144,8 @@ execute: |
     # shellcheck disable=SC2164
     pushd "$PROJECT_PATH"
     echo "Build the snap"
+    # Force use 16.04 packaging to build the snap
+    ln -sf packaging/ubuntu-16.04 debian
```

Comment:
Do you have a log from when this debian did not point to packaging/16.04? I wonder what happened there, a clean clone of the snapd tree has the symlink already set up correctly.